docs/resources/equinix_network_device.md

```diff
@@ -298,6 +299,9 @@ device configurations. See [Secondary Device](#secondary-device) below for more
 If not specified, default will be INTERNET-ACCESS
 * `project_id` - (Optional) Unique Identifier for the project resource where the device is scoped to.If you
 leave it out, the device will be created under the default project id of your organization.
+* `diverse_device_id` - (Optional) Unique ID of an existing device.
+Use this field to let Equinix know if you want your new device to be in a different location from any existing virtual 
+device. This field is only meaningful for single devices.
 
```

Comment:
the diverse_device_name field is not documented

Reply:
addressed them, pls check and let me know @displague

Reply:
The fields should be added here in the datasource documentation:
https://github.com/kpdhulipala/terraform-provider-equinix/blame/NFV-27633/docs/data-sources/equinix_network_device.md#L31

Reply:
sorry, I got it now, added them in data source equinix_network_device.md